src/SDPoptimize.jl

```diff
@@ -503,12 +505,13 @@ Returns :
 
 - controls (Array{Float64})
     the controls applied to the system at each time step
+
 """
 function sdp_forward_simulation(model::SPModel,
                   param::SDPparameters,
                   scenarios::Array{Float64,3},
-                  value::Array,
-                  display=true::Bool)
+                  V::Array,
+                  display=false::Bool)
 
     SDPmodel = build_sdpmodel_from_spmodel(model)
```

Comment:
As this might take time etiher duplicate the function or test with an if if SDP is already an SPmodel.
Alternatively the test could be conducted in build_sdpmodel_from_spmodel (if not already done).